omero/sysadmins/unix/server-centos6-ice36.rst

```diff
@@ -23,6 +22,10 @@ This guide describes how to install the **recommended** versions, not all
 the supported versions.
 This should be read in conjunction with :doc:`../version-requirements`.
 
+.. warning::
+
+   CentOS 6 is deprecated, CentOS 7 is preferable for new installations
```

Comment:
"See" should be a new sentence or separated with a semi-colon from the preceding text.